src/components/ComboBox/ComboBox.jsx

```diff
@@ -37,6 +38,8 @@ const defaultProps = {
   editOptionText: '-Create',
   hasMultiValue: false,
   items: [],
+  onInputChange: defaultFunction('onInputChange'),
```

Comment:
Is this required? It's not defined in proptypes above. 

We haven't used this helper outside of the table props before, but I'm not opposed to using it here for now. You might consider null or a noop though, see https://github.com/IBM/carbon-addons-iot-react/issues/1401